microsoft-edge/webview2/releasenotes.md

```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `prerelease package`.  
 
+## 1.0.674-prerelease  
+
+Release Date: Oct 19, 2020  
+
+[NuGet package][NuGetGallery1.0.674-prerelease] \| minimum Microsoft Edge version 88.0.674.0.
+
+#### General
+
+*   Added [WebView2 Group Policies][DeployedgeMicrosoftEdgeWebviewPolicies]. For more information on best practices, navigate to [group policies for WebView2][Webview2ConceptsEnterpriseGroupPoliciesForWebview2].
```

Comment:
We decided on Friday to break and change the naming of some policies, so let's delete the group policy entry in release notes for now.